### PR TITLE
[FIX] VIA Hex Generation Fix

### PR DESCRIPTION
**Pull request by Xelus22**
## Description

merge/um70:via did not previously compile as it was over the size limit. 
https://github.com/Xelus22/QMK-VIA-Hex/runs/2588296374?check_suite_focus=true#step:658:659

On AVR-GCC 8.3.0
```
Linking: .build/merge_um70_via.elf                                                                  [OK]
Creating load file for flashing: .build/merge_um70_via.hex                                          [OK]
Copying merge_um70_via.hex to qmk_firmware folder                                                   [OK]
Checking file size of merge_um70_via.hex                                                            [WARNINGS]

 * The firmware size is approaching the maximum - 28558/28672 (99%, 114 bytes free)
```
Travis uses 5.4.0

## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Compile under size by removing NKRO and enable `LAYER_STATE_8BIT`

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by fauxpark — APPROVED** [review]

File: keyboards/merge/um70/keymaps/via/rules.mk [hunk 1]
```diff
@@ -1,3 +1,5 @@
 VIA_ENABLE = yes
 WPM_ENABLE = yes
-LTO_ENABLE = yes
\ No newline at end of file
+LTO_ENABLE = yes
+
+NKRO_ENABLE = no
```
Comment (drashna) [inline keyboards/merge/um70/keymaps/via/rules.mk:5]: I'm not a fan of this.  NKRO is important to a lot of people.

File: keyboards/merge/um70/keymaps/via/config.h [hunk 2]
```diff
@@ -1,20 +1,23 @@
...
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
 #define RGBLIGHT_EFFECT_BREATHING
 #define RGBLIGHT_EFFECT_RAINBOW_MOOD
 #define RGBLIGHT_EFFECT_RAINBOW_SWIRL
 #define RGBLIGHT_EFFECT_SNAKE
+
+#define LAYER_STATE_8BIT
+
```
Comment (drashna) [inline keyboards/merge/um70/keymaps/via/config.h:23]:
```suggestion

/* Disable unused and unneeded features to reduce on firmware size */
#ifdef LOCKING_SUPPORT_ENABLE
#    undef LOCKING_SUPPORT_ENABLE
#endif
#ifdef LOCKING_RESYNC_ENABLE
#    undef LOCKING_RESYNC_ENABLE
#endif
```

**Review by drashna — CHANGES_REQUESTED** [review]
In fact, a better way to reduce the size would be to move the logo out of the keymap (it's using a raw oled image) and into a custom font file. 

the raw image is probably why it's over sized.

File: keyboards/merge/um70/keymaps/via/rules.mk (hunk 1 above)
Comment (Xelus22) [reply keyboards/merge/um70/keymaps/via/rules.mk:5]: I have tested after enabling NKRO, and removing `#define RGBLIGHT_EFFECT_SNAKE`, it compiles under at 184 bytes free.

```Linking: .build/merge_um70_via.elf                                                                  [OK]
Creating load file for flashing: .build/merge_um70_via.hex                                          [OK]
Copying merge_um70_via.hex to qmk_firmware folder                                                   [OK]
Checking file size of merge_um70_via.hex                                                            [WARNINGS]

 * The firmware size is approaching the maximum - 28488/28672 (99%, 184 bytes free)```

**Comment by Xelus22** [comment]
Closing this PR in favour of #12919

**Pull request closed without merging**